src/components/hmi_message_handler/test/hmi_message_adapter_test.cc

```diff
@@ -42,7 +42,7 @@ namespace test {
 namespace components {
 namespace hmi_message_handler_test {
 
```

Comment:
@dev-gh related indirectly due to changes in `mock_hmi_message_handler_settings.h` in #112